.github/workflows/dry-publish.js.yml

```diff
@@ -0,0 +1,30 @@
+# The objective of the dry-run job is to get a preview of the pending release.
+# Dry-run mode skips the following steps: prepare, publish, success and fail.
+# In addition to this it prints the next version and release notes to the console.
+
+# Note: The Dry-run mode verifies the repository push permission, even though nothing will be pushed.
+# The verification is done to help user to figure out potential configuration issues.
+name: Dry Publish
+
+
+on:
+  workflow_dispatch:
+
+jobs:
+  dry-publish:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
@abod-ibm are we using node14?

Reply:
@dventurait  Yeah, Node v14 is required for running semantic-release script.
![image](https://user-images.githubusercontent.com/67692574/151991667-bfc24ce9-f036-40c8-8bde-2fb53e0a34d0.png)